build_js.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+git submodule update --init --recursive
+
+DATA_DIR="$(pwd)/data/"
+
+mkdir -p build_js
+cd build_js
+
+cmake ../src \
+    -DBUILD_GUI_VIEWERS=ON \
+    -DBUILD_PYTHON_BINDINGS=OFF \
+    -DBUILD_DATATOOL=OFF \
+    -DBUILD_PTEX_SUPPORT=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_PREFIX_PATH="$EMSCRIPTEN" \
+    -DTARGET_GLES2=OFF \
+    -DCMAKE_TOOLCHAIN_FILE="../src/deps/corrade/toolchains/generic/Emscripten-wasm.cmake" \
+    -DCMAKE_INSTALL_PREFIX="." \
+    -DCMAKE_CXX_FLAGS="-s FORCE_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1 --preload-file $DATA_DIR/scene_datasets/habitat-test-scenes@/" \
+
+cmake --build . -- -j 4
+cmake --build . --target install -- -j 4
+
+echo "Done building."
+echo "Run:"
+echo "python -m http.server"
```

Comment:
`python3 -m http.server`? I got a lot of complaints about this from people who still suffer from Python 2 being the default on their system :)